cdc-mysql/src/bin/consumer.rs

```diff
@@ -64,7 +68,9 @@ async fn consume(
 
     // read read from producer and print to terminal
     while let Some(Ok(record)) = stream.next().await {
```

Comment:
this seems like duplicate of Fluvio CLI.  this should be moved into Fluvio CLI otherwise, remove it